NOTICE

```diff
@@ -0,0 +1,39 @@
+http4s
+Copyright 2013-2018 http4s.org
+Licensed under Apache License 2.0 (see LICENSE)
```

Comment:
We're technically supposed to include the license of all the projects we copied from.  I took the liberty of doing so while adding yet another project to copy and paste from.

A former employer's lawyer once told me we need to have the copyright boilerplate on all our source files.  I told him not while we were maintaining multiple branches.  But maybe now is a good time.  But that's a discussion for another ticket.